### PR TITLE
add geometa recipe

### PR DESCRIPTION
**Pull request by Marie59**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Marie59** [comment]
Hi again @dbast ! I did as you said and did 3 seperated PR. So here for geometa we have the same issue as previously that you said could be ignored. 
Thanks again for your help !

**Comment by Marie59** [comment]
@conda-forge/help-r, ready for review !

File: recipes/r-geometa/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+about:
+  home: https://github.com/eblondel/geometa/wiki
+  license: MIT
+  summary: Provides facilities to handle reading and writing of geographic metadata defined with
+    OGC/ISO 19115, 11119 and 19110 geographic information metadata standards, and encoded
+    using the ISO 19139 (XML) standard. It includes also a facility to check the validity
+    of ISO 19139 XML encoded metadata.
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
```
Comment (dbast) [inline recipes/r-geometa/meta.yaml:56]: as stated by the cran meta data
```suggestion
    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
    - LICENSE
```

**Comment by dbast** [comment]
ignoring the win failure due to dependencies as it is a noarch pkg  which builds and uploads on linux.

**Pull request merged**